src/test/java/gov/usgs/cida/pubs/busservice/ipds/IpdsProcessTest.java

```diff
@@ -194,7 +191,7 @@ public void okToProcessTest() {
 		newMpPub.setPublicationType(pubType);
 
 		//Good Dissemination (brand new)
-		assertTrue(ipdsProcess.okToProcess(ProcessType.DISSEMINATION, newMpPub, new MpPublication()));
+		assertFalse(ipdsProcess.okToProcess(ProcessType.DISSEMINATION, newMpPub, new MpPublication()));
```

Comment:
I don't understand what this testing? Dissemination is done elsewhere now.

Reply:
This is just testing that a dissemination will no longer be okToProcess in this manner.